### ISSUE TITLE
better color for line highlighting

### ISSUE THREAD

**Issue by rvg02010** [jnolis/synthwave85#4]
First of all, I'm obsessed, thank you for your Very Important Contribution.

If you have line highlighting enabled, the color is a muddy grey that doesn't fit well with the rest of the styling! Unsure what the JS style class is or what a better color would be. Screenshot below w/ the point of contention.

<img width="716" alt="Screen Shot 2019-05-06 at 11 40 57 AM" src="https://user-images.githubusercontent.com/13634103/57247184-ff271c00-6ff3-11e9-8301-6045ea2b2f5f.png">

**Comment by jnolis** [comment]
Oh good catch, that's something I missed from the base theme. I'll check it out!

**Comment by jnolis** [comment]
Okay, I have the color looking better, can you tell me what you think of this? If you like it I'll launch it (I never use this feature so I can't tell if it's too bright or dark).

You can test it yourself by uninstalling the old version and running:
```r
rstudioapi::addTheme("https://raw.githubusercontent.com/jnolis/synthwave85/active-line-issue/Synthwave85.rstheme", TRUE, TRUE, FALSE)
```

![image](https://user-images.githubusercontent.com/3317138/57260355-095b1180-7018-11e9-8a82-7ff20c97c63c.png)

**Comment by rvg02010** [comment]
finally had a moment to check it out--perfect! thank youuuuu

**Comment by jnolis** [comment]
Closed via commit #f0305bbb !
